doc/user/content/releases/_index.md

```diff
@@ -152,12 +151,16 @@ See also Upgrade Notes for release specific notes.
 
 {{</ tab >}} {{</ tabs >}}
 
-### Upgrade notes for v26.0.0
+#### Upgrade notes for v26.0.0
 
 {{< include-md file="shared-content/self-managed/upgrade-notes/v26.0.md" >}}
 
 See also [General notes for upgrades](/installation/#general-notes-for-upgrades).
 
+## Self-managed release versions
```

Comment:
We could also move it to the top in case we don't write anything for the first weekly release ... but ... will let you make the call.